### PR TITLE
add pangolin-parachain

### PR DESCRIPTION
**Pull request by xiaoch05**

### CODE REVIEW

File: apollo/src/substrate2parachain/substrate2parachain.service.ts [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+      });
+
+      if (unconfirmedRecords.length == 0) {
+        if (isLock) {
+          this.needSyncLockConfirmed = false;
+        } else {
+          this.needSyncBurnConfirmed = false;
+        }
+        return;
+      }
```
Comment (sxlwar) [inline apollo/src/substrate2parachain/substrate2parachain.service.ts:150]:
```js
if (isLock) {
          this.needSyncLockConfirmed = false;
        } else {
          this.needSyncBurnConfirmed = false;
        }
```
this.needSyncBurnConfirmed = false;

File: apollo/src/substrate2parachain/substrate2parachain.service.ts [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+
+      const targetNonces = [];
+
+      for (const record of unconfirmedRecords) {
+        targetNonces.push('"' + record.nonce + '"');
```
Comment (sxlwar) [inline apollo/src/substrate2parachain/substrate2parachain.service.ts:155]: `targetNonces.push(record.nonce.toString());` ?

File: apollo/src/substrate2parachain/substrate2parachain.service.ts [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+        variables: null,
+      });
+
+      const nodes = res.data?.data?.s2sEvents?.nodes;
+      const timezone = new Date().getTimezoneOffset() * 60;
+
+      if (nodes && nodes.length > 0) {
+        var updated = 0;
+        for (const node of nodes) {
+          if (node.result == 0) {
```
Comment (sxlwar) [inline apollo/src/substrate2parachain/substrate2parachain.service.ts:171]: 用 ===吧，js 里 == 不安全

**Review by sxlwar — CHANGES_REQUESTED** [review]

File: apollo/src/substrate2parachain/substrate2parachain.service.ts (hunk 2 above)
Comment (xiaoch05) [reply apollo/src/substrate2parachain/substrate2parachain.service.ts:155]: this will put into subql request.

**Pull request merged**